src/scripts/main.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
+const div = document.createElement('div');
+
+div.style.cssText = `display:flex; flex-direction: column; row-gap: 10px; padding-left: 10px`;
+document.body.append(div);
+
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, (e) => {
```

Comment:
Why do you add a listener in the `waitFor` function? It's confusing.
With a current name it should just wait and doesn't know anything about elements and listeners